tools/generator/Java.Interop.Tools.Generator.Importers/XmlApiImporter.cs

```diff
@@ -116,6 +116,10 @@ public static ClassGen CreateClass (XElement pkg, XElement elem, CodeGenerationO
 					 !options.SupportNestedInterfaceTypes
 			};
 
+			if (elem.Attribute ("skipInvokerMethods")?.Value is string skip)
+				foreach (var m in skip.Split (new char [] { ',', ' ' }))
```

Comment:
This should use `StringSplitOptions.RemoveEmptyEntries`: https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-7.0#system-string-split(system-char()-system-stringsplitoptions)

Reply:
Should also add `\n` and `\r` in there as well.